app/src/androidTest/java/com/github/se/cyrcle/di/mocks/MockAddressRepository.kt

```diff
@@ -16,11 +16,17 @@ class MockAddressRepository @Inject constructor() : AddressRepository {
           postcode = "Mock Postcode",
           country = "Mock Country")
 
-  override fun search(query: String, onSuccess: (Address) -> Unit, onFailure: (Exception) -> Unit) {
+  private val mockAddress2 = mockAddress.copy(city = "Mock City 2")
```

Comment:
Why did you add another MockAddress here? Does the code not function if the list only has one element in it?

Reply:
The idea here is to test the feature `SuggestionMenu`. This menu triggers only when the number of suggestion is superior to 1. If it is 1 suggestion only then the camera is automatically recentered to this location.

Reply:
Ahh yes, indeed, makes sense!